test/integration/bankHolidayAlert.js

```diff
@@ -0,0 +1,67 @@
+const chai = require('chai');
+const chaiHttp = require('chai-http');
+const cheerio = require('cheerio');
+const nock = require('nock');
+
+const constants = require('../../app/lib/constants');
+const getSampleResponse = require('../resources/getSampleResponse');
+const iExpect = require('../lib/expectations');
+const server = require('../../server');
+
+const expect = chai.expect;
+
+chai.use(chaiHttp);
+
+const resultsRoute = `${constants.SITE_ROOT}/results`;
+const numberOfOpenResults = constants.numberOfOpenResults;
+const numberOfNearbyResults = constants.numberOfNearbyResultsToRequest;
+const yourLocation = constants.yourLocation;
+
+describe('The bank holiday alert messaging', () => {
+  after('clean nock', () => {
+    nock.cleanAll();
+  });
+
+  afterEach('reset DATE', () => {
+    process.env.DATE = '2017-12-12';
+  });
+
+  describe('on the day of the bank holiday', () => {
+    before('set DATE', () => {
+      process.env.DATE = '2017-12-25';
+    });
+
+    it('should show a message about the bank holiday for each open result', (done) => {
```

Comment:
Does this mean that every pharmacy that is open in the list of results should have a message next to it, or only the one in the 'nearest open' section? 

From the implementation it looks like it is just the top section, as it's only looking for one message, which is the number of configured results in that section.

Reply:
I can see the confusion I'll try to make it clearer. Every open org should get the message and in fact this is now extended to be every org that is open today.